addon/components/bs-modal/dialog.hbs

```diff
@@ -13,9 +13,15 @@
   {{did-insert this.getOrSetTitleId}}
   {{did-insert this.setInitialFocus}}
 >
-  <div class="modal-dialog {{this.sizeClass}} {{if @centered "modal-dialog-centered"}} {{if @scrollable "modal-dialog-scrollable"}}" role="document">
-    <div class="modal-content"
-     tabindex="-1"
+  <div class="modal-dialog {{this.sizeClass}} {{if @centered "modal-dialog-centered"}} {{if @scrollable "modal-dialog-scrollable"}} 
+  {{#if @fullscreen}}
+    {{#if (bs-eq @fullscreen true)}}
+      modal-fullscreen
+    {{else}}
+      modal-fullscreen-{{@fullscreen}}-down
+    {{/if}}
+  {{/if}}" role="document">
+    <div class="modal-content" tabindex="-1"
```

Comment:
Any specific reason for which you removed the linebreak before `tabindex`? Or was that just by mistake?

Reply:
I think this is my linter or beautifyer addon !

Reply:
```suggestion
    <div class="modal-content"
      tabindex="-1"
```